js-exercises/sum-primes/sumPrimes.js

```diff
@@ -0,0 +1,24 @@
+function sumPrimes(num) {
+  let prime = [];
+
+  for (var i = 2; i <= num; i++) {
```

Comment:
Use `let` in C style for loops, `let` is block scoped.

Reply:
Also, try making a `range` generator and using it.